test_runner/regress/test_bad_connection.py

```diff
@@ -0,0 +1,60 @@
+import random
+import time
+
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnvBuilder
+
+
+def test_stress_with_bad_connection(neon_env_builder: NeonEnvBuilder):
```

Comment:
nit: can you make the test names a bit more specific?  Something like `test_compute_pageserver_connection_stress`, so that it's a bit more obvious what kind of  connections we're talking about.

Reply:
Changed